tests/conftest.py

```
@@ -11326,31 +11326,114 @@ def teardown():
                 if "NotFound" in str(e):
                     log.warning(f"Cloned VM {cloned_vm.name} was already deleted.")
                 else:
                     raise
 
-            run_oc_command(
-                cmd=f"delete pvc {cloned_vm.pvc_name}", namespace=cloned_vm.namespace
-            )
-            log.info(f"Cloned VM PVC {cloned_vm.pvc_name} deleted")
+            except TimeoutError:
+                log.warning(
+                    f"Timed out waiting for cloned VM {cloned_vm.name} deletion"
```

Ruff (0.15.17)

[warning] Logging statement uses f-string

(G004)